gulp-tasks/publish-cdn.js

```diff
@@ -54,6 +59,11 @@ gulp.task('publish-cdn:generate-from-tags', async () => {
 });
 
 gulp.task('publish-cdn:temp-v3', async () => {
+  // Let's force this to always be fresh - in case we run it outside of
+  // gulp publish
+  fs.removeSync(
```

Comment:
Since this is already an `async` function, could you switch this to:

```js
await fs.remove(...);
```

And switch the other `fs` method calls in this function, too? I know it doesn't make a difference in the current order of operations, but getting into the habit of preferring `await fs.something()` should guard against recurrences of https://github.com/GoogleChrome/workbox/pull/883